File: Common/InMotionGIT.Common - Copy/Enumerations/EnumCache.vb
Imports System.ComponentModel

Namespace Enumerations

    <DefaultValue(EnumCache.None)>
    Public Enum EnumCache
        None = 0
        CacheWithFullParameters = 1
        CacheWithCommand = 2
        CacheOnDemand = 3
    End Enum

End Namespace

File: Common/InMotionGIT.Common.Domain/Enumerations/EnumCache.cs
using System.ComponentModel;

namespace InMotionGIT.Common.Domain.Enumerations;


[DefaultValue(None)]
public enum EnumCache
{
    None = 0,
    CacheWithFullParameters = 1,
    CacheWithCommand = 2,
    CacheOnDemand = 3
}